makefiles/tools/uniflash.inc.mk

```
@@ -24,8 +24,10 @@ else
   RESET ?= $(UNIFLASH_PATH)/uniflash.sh
   RESET_FLAGS ?= -ccxml $(UNIFLASH_CONFIG_CCXML) -reset
 endif
+
+export CCS_PATH ?= "CCS_PATH unconfigured"
```

```suggestion
CCS_PATH ?= "CCS_PATH unconfigured"
```
Note that the UNIFLASH_PATH variable above should not be exported as well (could be changed in a separate commit).